walk-in-interview/src/main/webapp/log-in/business/script.js

```diff
@@ -55,4 +62,51 @@ backButton.addEventListener('click', (_) => {
 const submitButton = document.getElementById('submit');
 submitButton.addEventListener('click', (_) => {
   // TODO(issue/78): send the account & password input to the firebase auth related stuff
+  const account = document.getElementById('account-input').value;
+
+  if (account === '') {
+    setErrorMessage(/* errorMessageElementId= */'error-message',
+        /* msg= */ STRINGS['empty-email'],
+        /* includesDefault= */false);
+    return;
+  }
+
+  const password = document.getElementById('password-input').value;
+
+  // Resets the error (there might have been an error msg from earlier)
+  setErrorMessage(/* errorMessageElementId= */'error-message',
+      /* msg= */ '',
+      /* includesDefault= */false);
+
+  signIntoBusinessAccount(account, password)
+      .catch((error) => {
+        switch(error.code) {
```

Comment:
Maybe abstract these to another method? e.g showErrorMessageFromCode(errorCode) ?